proto/evmos/inflation/v1/genesis.proto

```diff
@@ -17,6 +17,11 @@ message GenesisState {
   string epoch_identifier = 3;
   // number of epochs after which inflation is recalculated
   int64 epochs_per_period = 4;
+  // fraction of the staking tokens which are currently bonded
+  string bonded_ratio = 5 [
```

Comment:
@fedekunze We need to store it because we calculate the `epochMintProvision` once at the end of the period. If we get it from the staking keeper the `epochMintProvision` will be recalculated with a different bondedRatio, when a node runs the genesis at a time other than the epoch end, right?

Reply:
> will be recalculated with a different bondedRatio

I didn't get this part, can you elaborate?